xbmc/rendering/gles/RenderSystemGLES.cpp

```diff
@@ -460,7 +460,7 @@ void CRenderSystemGLES::GetViewPort(CRect& viewPort)
 }
 
 // FIXME make me const so that I can accept temporary objects
-void CRenderSystemGLES::SetViewPort(CRect& viewPort)
+void CRenderSystemGLES::SetViewPort(const CRect& viewPort)
```

Comment:
Comment above no longer makes sense?

Reply:
Haha didn't even notice that this completed an 8-year-old FIXME.

Reply:
what took you so long?